src/simple-tools.ts

```diff
@@ -18,7 +18,7 @@ function cloneSimpleObject(obj: any) {
  */
 function addUniqueToArray(array: any[], obj: any) {
   let found = false;
-  for (const i in array) {
+  for (const i of array) {
```

Comment:
Wouldn't iterate over the values instead of indexes?

Reply:
Yes, I'll change that.